File: CS/DXSampleCS/DXSample.Module/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.ExpressApp;
using DevExpress.Persistent.BaseImpl;

namespace DXSample.Module {
    public class Updater : ModuleUpdater {
        public Updater(ObjectSpace objectSpace, Version currentDBVersion) : base(objectSpace, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            CreateUser("Sam", "", true);
            CreateUser("John", "", false);
            CreateUser("Paul", "", false);
        }
        private void CreateUser(string name, string password, bool isAdministrator) {
            BasicUser BasicUser = ObjectSpace.FindObject<BasicUser>(new BinaryOperator("UserName", name));
            if (BasicUser == null) {
                BasicUser = ObjectSpace.CreateObject<BasicUser>();
                BasicUser.UserName = name;
                BasicUser.FullName = name;
            }
            BasicUser.IsAdministrator = isAdministrator;
            BasicUser.SetPassword(password);
            BasicUser.Save();
        }
    }
}

File: VB/VB/DXSample.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.Persistent.BaseImpl

Namespace DXSample.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal session As Session, ByVal currentDBVersion As Version)
			MyBase.New(session, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			CreateUser("Sam", "", True)
			CreateUser("John", "", False)
			CreateUser("Paul", "", False)
		End Sub
		Private Sub CreateUser(ByVal name As String, ByVal password As String, ByVal isAdministrator As Boolean)
			Dim BasicUser As BasicUser = Session.FindObject(Of BasicUser)(New BinaryOperator("UserName", name))
			If BasicUser Is Nothing Then
				BasicUser = New BasicUser(Session)
				BasicUser.UserName = name
				BasicUser.FullName = name
			End If
			BasicUser.IsAdministrator = isAdministrator
			BasicUser.SetPassword(password)
			BasicUser.Save()
		End Sub
	End Class
End Namespace
